composer.json

```diff
@@ -122,5 +124,7 @@
             "dev-develop": "4.4.x-dev"
         },
         "contao-manager-plugin": "Contao\\CoreBundle\\ContaoManager\\Plugin"
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
This should not be necessary if you suffix the requirement with `@dev`. If that's an issue with an included package, simply require it in the root as well (with `@dev`).